### ISSUE TITLE
Investigate what screenreaders do for their image description in the browser (@Sabrina )

### ISSUE THREAD

**Issue by Sabrina-Knappe** [Shared-Reality-Lab/IMAGE-browser#262]

**Comment by Sabrina-Knappe** [comment]
Determine what screendreaders currently do for their image ID. Please close this issue with a quick rundown.

**Comment by Cybernide** [comment]
Moving this to next sprint

**Comment by Cybernide** [comment]
### JAWS
Upon navigating (using keys) to a graphic, the screen reader says "Image: " followed by the alt text, if available.
### NVDA
Upon navigating (used mouse) to a graphic, the screen reader just reads the alt text, if no alt text is available, it'll just say "no photo description available"

**Comment by jeffbl** [comment]
@Cybernide @Sabrina-Knappe what needs to happen to complete this work item? Is this information being factored into design decisions for another work item, or is there move investigation needed here?

**Comment by Cybernide** [comment]
@Sabrina-Knappe could you provide the output for Voiceover so we can close this issue and report this somewhere?

**Comment by Cybernide** [comment]
See entry for Voiceover in previous comment

**Comment by Cybernide** [comment]
Can we close this issue now that we have results from the top 3 screenreaders people use? @Sabrina-Knappe @jeffbl

**Comment by Cybernide** [comment]
Closing @jeffbl
